packages/vxrn/src/nativePlugin.ts

```diff
@@ -71,22 +69,7 @@ export function nativePlugin(options: {
       config.optimizeDeps.esbuildOptions.loader ??= {}
       config.optimizeDeps.esbuildOptions.loader['.js'] = 'jsx'
 
-      config.optimizeDeps.esbuildOptions.plugins.push({
-        name: 'react-native-assets',
-        setup(build) {
-          build.onResolve(
-            {
-              filter: /\.(png|jpg|gif|webp)$/,
-            },
-            async ({ path, namespace }) => {
-              return {
-                path: '',
-                external: true,
-              }
-            }
-          )
-        },
-      })
+      // TODO: move `@vxrn/react-native-prebuilt` logic into `config.optimizeDeps`, to support various React Native versions, not just one.
```

Comment:
we don't need to make it right now, it's not the easiest thing to do. let's leave it for future perf optimisation.